### PR TITLE
Updated email address

### PR DESCRIPTION
**Pull request by bhasin85**
Related [PR](https://github.com/aodn/chef-private/pull/2032)

### CODE REVIEW

**Comment by bhasin85** [comment]
# Do Not Merge

Testing

File: cookbooks/imos_webapps/templates/default/portal/Portal.groovy.erb [hunk 1]
```diff
@@ -133,7 +133,8 @@ portal {
     }
```
Comment (pmbohm) [inline cookbooks/imos_webapps/templates/default/portal/Portal.groovy.erb]: Its worth changing line 132 as well
`fromAddress = "<%= @custom_parameters['system_email_from_address'] || 'info@emii.org.au' %>"`
to
`fromAddress = "<%= @custom_parameters['system_email_from_address'] || 'info@aodn.org.au' %>"`

https://github.com/aodn/chef/blob/5f2c6f2095d0b30c5ab496a161c76dec40bcbde5/cookbooks/imos_webapps/templates/default/portal/Portal.groovy.erb#L132

**Comment by pmbohm** [comment]
By the way the name of your branch is confusing. `issue_409`
Maybe you mean `backlog_409`

File: cookbooks/imos_webapps/templates/default/portal/Portal.groovy.erb (hunk 1 above)
Comment (bhasin85) [inline cookbooks/imos_webapps/templates/default/portal/Portal.groovy.erb]: Done :+1:

**Pull request merged**